packages/ckeditor5-core/src/editor/editor.ts

```diff
@@ -268,19 +268,20 @@ export default abstract class Editor extends ObservableMixin() {
 	 *
 	 * Usually, not to be used directly. See the static {@link module:core/editor/editor~Editor.create `create()`} method.
 	 *
-	 * @param config The editor configuration.
+	 * @param providedConfig The editor configuration.
```

Comment:
Do we want to change the API documentation because we wanted to use the `config` name for local variable?

Reply:
I wasn't sure about this change too. I reverted it, so that we don't change our API docs.